testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -115,6 +115,29 @@ def LAMMPSDATAWriter(request, tmpdir_factory):
     return u, u_new
 
 
+@pytest.fixture(params=[
+    LAMMPSdata,
+    LAMMPSdata_mini,
+    LAMMPScnt,
+    LAMMPShyd,
```

Comment:
From the code coverage comment above, it seems that none of those covers the case where there is no charge. Can we add one such case?